### ISSUE TITLE
Boomer Gamemode

### ISSUE THREAD

**Issue by SireTurret** [HippieStation/HippieStationdeprecated2020#11896]
[Title]: # Boomer Gamemode

[Implementation]: # Basically, there's 3 boomers randomly selected from the crewmembers. They have some guns from the classic DOOM and see every other player as a monster from DOOM. They DO NOT have access to medical items, but rather they have their own medical item - the boomer juice. You can refill the boomer juice by running someone over with a lawnmower or killing someone brutally using the guns from DOOM.

EDIT: (Suggested by Maroon)
"i'd also like something that builds up to the boomers getting doom weapons like fueling up from enough boomer juice made from random chemicals or completing objectives then yes, we can make this happen and set us apart from the other servers in a hippie fashion!"

Even though this may seem OP, the crewmembers in return get something zoomer-like - for an example, a shield potion from Fort Night.

[Why]: # It seems like a fun idea. If you have any suggestions, post 'em here.

[Resources]: # honestly no idea, the lawnmower is already in, the guns need to be sprited and made probably

# Be sure to ask some maintainer to label this issue as a feature. Either ping them on our discord or PM them on our forum. Thank you for sharing your ideas!
@HippieStation/maintainers

**Comment by Majick1** [comment]
+1 sure

**Comment by cb583** [comment]
+1
epic idea

**Comment by TrickyDickRichard** [comment]
+1 sounds fun

**Comment by G-erma** [comment]
+1 yes

**Comment by waterwater1212** [comment]
Boomers Vs. Zoomers

**Comment by ghost** [comment]
-1 retarded

**Comment by SireTurret** [comment]
> -1 retarded

zoomer

**Comment by ghost** [comment]
SHUT THE FUCK YOU R*TARD (censored to avoid github ban), YOU DON'T KNOW WHAT'S GOOD AND WHAT'S BAD, YOU'RE JUST A FUCKING IDIOT WHO DOESN'T KNOW CULTURE FROM DIARRHEA

**Comment by SireTurret** [comment]
> SHUT THE FUCK YOU R*TARD (censored to avoid github ban), YOU DON'T KNOW WHAT'S GOOD AND WHAT'S BAD, YOU'RE JUST A FUCKING IDIOT WHO DOESN'T KNOW CULTURE FROM DIARRHEA

Mr. Seethe

**Comment by Maroonenahmoron** [comment]
It’s really out of line of the gamemodes we have, not taking into consideration of the station as a whole. If there is something that builds up to the boomers getting doom weapons like fueling up from enough boomer juice made from random chemicals or completing objectives then yes, we can make this happen and set us apart from the other servers in a hippie fashion!

**Comment by ghost** [comment]
yeah make boomers just occasionally fart audibly in a 3x3 radius while not yet powered,  gradually getting stronger and annoyed at the zoomers  until they can pull out their weapons from under a tile with a crowbar.  sipping soda accelerates the power up,  and you dont power up unless you watch other people (zoomers) and get mad

**Comment by SireTurret** [comment]
> It’s really out of line of the gamemodes we have, not taking into consideration of the station as a whole. If there is something that builds up to the boomers getting doom weapons like fueling up from enough boomer juice made from random chemicals or completing objectives then yes, we can make this happen and set us apart from the other servers in a hippie fashion!

Seems like a good idea, I'll add this to the description.



> yeah make boomers just occasionally fart audibly in a 3x3 radius while not yet powered, gradually getting stronger and annoyed at the zoomers until they can pull out their weapons from under a tile with a crowbar. sipping soda accelerates the power up, and you dont power up unless you watch other people (zoomers) and get mad

make zoomers forcefully dab and whenever they do that make boomers see that as a giant red text

**Comment by cris5311** [comment]
+1

**Comment by SireTurret** [comment]
my fellow boomers let's hang steamp0rt and laugh at him for being a soylent chugging fat but skinny zoomer

**Comment by Manly-Dwarf** [comment]
Could just add this antagonist as a secret one to Dynamic rather than making it its own mode. That's the most likely way you'll ever see it.

**Comment by SireTurret** [comment]
okay, after some discussion it was decided that this idea will be transformed into a very rare dynamic gamemode antag instead of this being it's own

**Comment by MrHeroKong** [comment]
They should also get a '69 Pontiac GTO that works like a lawn mower/clown car hybrid, all the time putting CO2 into the air at an alarming rate, causing suffocation.
